wordpress-mobile/gutenberg-mobile
# RichText State updates

Fix state updates on RichText.

How to test:
 - On Heading blocks, change the type of heading and see if the block content updates imediately
 - On Paragraph and Heading block select text and apply a style (bold) see if it's applied visually and then switch to HTML and check in HTML generated is correct.
